example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -198,16 +203,19 @@ describe('The S3 Ingest Granules workflow', () => {
     });
 
     afterAll(async () => {
-      await s3().deleteObject({ Bucket: files[0].bucket, Key: files[0].filepath }).promise();
-      await s3().deleteObject({ Bucket: files[1].bucket, Key: files[1].filepath }).promise();
-      await s3().deleteObject({ Bucket: files[3].bucket, Key: files[3].filepath }).promise();
+      await Promise.all([
+        s3().deleteObject({ Bucket: files[0].bucket, Key: files[0].filepath }).promise(),
+        s3().deleteObject({ Bucket: files[1].bucket, Key: files[1].filepath }).promise(),
+        s3().deleteObject({ Bucket: files[3].bucket, Key: files[3].filepath }).promise()
+      ]);
     });
 
-    it('has a payload with correct buckets and filenames and filesizes', () => {
+    it('has a payload with correct buckets, filenames, filesizes, and duplicate reporting', () => {
       files.forEach((file) => {
         const expectedFile = expectedPayload.granules[0].files.find((f) => f.name === file.name);
         expect(file.filename).toEqual(expectedFile.filename);
         expect(file.bucket).toEqual(expectedFile.bucket);
+        expect(file.duplicate_found).toBe(expectedFile.duplicate_found);
```

Comment:
This feels like what @bilts had mentioned in the architecture meeting this week. Namely, running one workflow and then releasing a barrage of assertions against it. I'm a fan.